sources/javascripts/concat/global/main.js

```diff
@@ -436,6 +436,19 @@
       }
     }
 
+    var handleProductCountSync = function (e) {
```

Comment:
Pisiasi, aga seda `e` argumenti sa siin vist ei kasuta? Kui nii, siis ei ole seda siia vaja.
```suggestion
    var handleProductCountSync = function () {
```